docs/development_guide.md

```diff
@@ -159,6 +159,14 @@ After having pushed the tag, the following actions are performed:
 - A Docker image containing the installed Python distribution is built and pushed to Quay.
 - A GitHub release draft is created, which must be manually published.
 
+## Adding support for a new IBM Cloud Pak for Data 4.0.x release
+
+To add support for a new IBM Cloud Pak for Data 4.0.x release, perform the following steps:
+
+- Update `CloudPakForDataManager._IBM_CLOUD_PAK_FOR_DATA_VERSION`
```

Comment:
Please indicate the file that one can locate the `CloudPakForDataManager._IBM_CLOUD_PAK_FOR_DATA_VERSION`